src/main/java/io/vertx/rabbitmq/BasicProperties.java

```diff
@@ -0,0 +1,127 @@
+package io.vertx.rabbitmq;
+
+import io.vertx.codegen.annotations.CacheReturn;
+import io.vertx.codegen.annotations.GenIgnore;
+import io.vertx.codegen.annotations.VertxGen;
+
+import java.util.Date;
+import java.util.Map;
+
+/**
+ * Similar to {@link com.rabbitmq.client.BasicProperties}
+ */
+@VertxGen
+public interface BasicProperties {
+
+  /**
+   * Retrieve the value in the contentType field.
```

Comment:
the notion of field is internal to the implementation, we need to say what it actually returns, same for other accessors

Reply:
Agree, done